src/utils/aepp-wallet-communication/rpc/aepp-rpc.js

```diff
@@ -151,6 +156,26 @@ export default Ae.compose({
       })
       return this.sendConnectRequest()
     },
+    /**
+     * Connect to node
+     * @function bridgeNode
+     * @instance
+     * @return {void}
+     */
+    async bridgeNode () {
+      if (!this.rpcClient || !this.rpcClient.isConnected()) throw new NoWalletConnectedError('You are not connected to Wallet')
+      const walletNode = await this.sendBridgeRequest()
+      if (!walletNode.url) {
+        throw new Error('Missing node url')
+      } else {
+        const node = await Node({
```

Comment:
```suggestion
      }
      const node = await Node({
```
don't do extra nesting